Instructions for `auth-astro` results in build error

### 📚 Subject area/topic

astro-auth

### 📋 Page(s) affected (or suggested, for new content)

https://docs.astro.build/en/guides/authentication/#authjs

### 📋 Description of content that is out-of-date or incorrect

The instructions results in this build error:
```
❯ npm run build

> @example/basics@0.0.1 build
> astro build

16:55:51 [ERROR] [astro-auth] No Adapter found, please make sure you provide one in your Astro config
16:55:51 [ERROR] [astro-auth] An unhandled error occurred while running the "astro:config:setup" hook
Cannot read properties of undefined (reading 'name')
  Location:
    /home/projects/withastro-astro-ewbcra9m/node_modules/auth-astro/src/integration.ts:35:29
  Stack trace:
    at astro:config:setup (/home/projects/withastro-astro-ewbcra9m/node_modules/auth-astro/src/integration.ts:35:29)
    at withTakingALongTimeMsg (file:///home/projects/withastro-astro-ewbcra9m/node_modules/astro/dist/integrations/hooks.js:62:18)
    at Module.runHookConfigSetup (file:///home/projects/withastro-astro-ewbcra9m/node_modules/astro/dist/integrations/hooks.js:171:41)
    at async AstroBuilder.run (file:///home/projects/withastro-astro-ewbcra9m/node_modules/astro/dist/core/build/index.js:198:23)
    at async build (file:///home/projects/withastro-astro-ewbcra9m/node_modules/astro/dist/cli/build/index.js:44:3)
```
It seems the instructions are missing a number of additional steps or requirements that must be met first, in which case it should be pointed out in the document.

### 🖥️ Reproduction in StackBlitz (if reporting incorrect content or code samples)

1. Go to https://stackblitz.com/edit/withastro-astro-ewbcra9m?file=auth.config.ts
2. Execute `npm run build` in terminal

--- a/src/content/docs/en/guides/authentication.mdx
+++ b/src/content/docs/en/guides/authentication.mdx
@@ -25,7 +25,7 @@ Auth.js is a framework agnostic solution for authentication. A community framewo
 
 ### Installation
 
-Use the `astro add` command for your preferred package manager to add the `auth-astro` integration.
+Use the `astro add` command for your preferred package manager to add the `auth-astro` integration to an Astro project configured with a [server adapter](/en/guides/on-demand-rendering/#server-adapters) for on-demand rendering.
 
 <PackageManagerTabs>
   <Fragment slot="npm">
@@ -69,12 +69,13 @@ To install `auth-astro` manually, install the required package for your package 
 
 Then, apply the integration to your `astro.config.*` file using the `integrations` property:
 
-```ts title="astro.config.mjs" ins={2,6}
+```ts title="astro.config.mjs" ins={2,7}
 import { defineConfig } from 'astro/config';
 import auth from 'auth-astro';
 
 export default defineConfig({
   // ...
+  adapter: netlify(),
   integrations: [auth()],
 });
 ```
@@ -198,7 +199,7 @@ Follow the [Better Auth Astro Guide](https://www.better-auth.com/docs/integratio
 
 ### Usage
 
-Better Auth offers a `createAuthClient` helper for various frameworks, including Vanilla JS, React, Vue, Svelte, and Solid. 
+Better Auth offers a `createAuthClient` helper for various frameworks, including Vanilla JS, React, Vue, Svelte, and Solid.
 
 For example, to create a client for React, import the helper from `'better-auth/react'`:
 
@@ -272,7 +273,7 @@ You can then use the `auth` object to get the user's session data in your server
 import { auth } from "../../../lib/auth"; // import your Better Auth instance
 
 export const prerender = false; // Not needed in 'server' mode
- 
+
 const session = await auth.api.getSession({
 	headers: Astro.request.headers,
 });
@@ -286,7 +287,7 @@ You can also use the `auth` object to protect your routes using middleware. The 
 ```ts title="src/middleware.ts"
 import { auth } from "../../../auth"; // import your Better Auth instance
 import { defineMiddleware } from "astro:middleware";
- 
+
 export const onRequest = defineMiddleware(async (context, next) => {
 	const isAuthed = await auth.api
 		.getSession({

--- a/src/content/docs/en/guides/authentication.mdx
+++ b/src/content/docs/en/guides/authentication.mdx
@@ -273,6 +273,7 @@ You can then use the `auth` object to get the user's session data in your server
 import { auth } from "../../../lib/auth"; // import your Better Auth instance
 
 export const prerender = false; // Not needed in 'server' mode
+
 const session = await auth.api.getSession({
 	headers: Astro.request.headers,
 });
@@ -286,6 +287,7 @@ You can also use the `auth` object to protect your routes using middleware. The 
 ```ts title="src/middleware.ts"
 import { auth } from "../../../auth"; // import your Better Auth instance
 import { defineMiddleware } from "astro:middleware";
+
 export const onRequest = defineMiddleware(async (context, next) => {
 	const isAuthed = await auth.api
 		.getSession({

--- a/src/content/docs/en/guides/authentication.mdx
+++ b/src/content/docs/en/guides/authentication.mdx
@@ -273,7 +273,7 @@ You can then use the `auth` object to get the user's session data in your server
 import { auth } from "../../../lib/auth"; // import your Better Auth instance
 
 export const prerender = false; // Not needed in 'server' mode
-
+ 
 const session = await auth.api.getSession({
 	headers: Astro.request.headers,
 });
@@ -287,7 +287,7 @@ You can also use the `auth` object to protect your routes using middleware. The 
 ```ts title="src/middleware.ts"
 import { auth } from "../../../auth"; // import your Better Auth instance
 import { defineMiddleware } from "astro:middleware";
-
+ 
 export const onRequest = defineMiddleware(async (context, next) => {
 	const isAuthed = await auth.api
 		.getSession({

--- a/src/content/docs/en/guides/authentication.mdx
+++ b/src/content/docs/en/guides/authentication.mdx
@@ -199,7 +199,7 @@ Follow the [Better Auth Astro Guide](https://www.better-auth.com/docs/integratio
 
 ### Usage
 
-Better Auth offers a `createAuthClient` helper for various frameworks, including Vanilla JS, React, Vue, Svelte, and Solid.
+Better Auth offers a `createAuthClient` helper for various frameworks, including Vanilla JS, React, Vue, Svelte, and Solid. 
 
 For example, to create a client for React, import the helper from `'better-auth/react'`:
 
@@ -273,7 +273,6 @@ You can then use the `auth` object to get the user's session data in your server
 import { auth } from "../../../lib/auth"; // import your Better Auth instance
 
 export const prerender = false; // Not needed in 'server' mode
-
 const session = await auth.api.getSession({
 	headers: Astro.request.headers,
 });
@@ -287,7 +286,6 @@ You can also use the `auth` object to protect your routes using middleware. The 
 ```ts title="src/middleware.ts"
 import { auth } from "../../../auth"; // import your Better Auth instance
 import { defineMiddleware } from "astro:middleware";
-
 export const onRequest = defineMiddleware(async (context, next) => {
 	const isAuthed = await auth.api
 		.getSession({

